network/meson.build

```diff
@@ -2,17 +2,17 @@
 
 if conf.get('ENABLE_NETWORKD') == 1
         install_data(
-                '80-6rd-tunnel.network',
-                '80-auto-link-local.network.example',
-                '80-container-host0.network',
-                '80-container-vb.network',
-                '80-container-ve.network',
-                '80-container-vz.network',
-                '80-ethernet.network.example',
-                '80-vm-vt.network',
-                '80-wifi-adhoc.network',
-                '80-wifi-ap.network.example',
-                '80-wifi-station.network.example',
+                '90-6rd-tunnel.network',
+                '90-auto-link-local.network.example',
+                '90-container-host0.network',
+                '90-container-vb.network',
+                '90-container-ve.network',
+                '90-container-vz.network',
+                '99-ethernet.network.example',
+                '90-vm-vt.network',
+                '90-wifi-adhoc.network',
+                '90-wifi-ap.network.example',
+                '90-wifi-station.network.example',
```

Comment:
hmm, so i get this right, you are swapping the two levels? i.e previous generated was at 90 and static at 80? and now you make it the opposite?

in a way this is an api break, since people might have masked things, dunno.

i think we should still do this, but let's maybe at least not break both sides, but only one.

hence maybe leave the static ones at 80, but move the generated ones to 70?

Reply:
also, should have a quick note in NEWS incompat section, that this moved